### DISCUSSION TITLE
在识别中，如何指定不同的识别模型来识别不同的区域

### DISCUSSION THREAD

**Discussion by zz1988** [PaddlePaddle/PaddleOCR#12188] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu
- 版本号/Version：Paddle：2.5.0  PaddleOCR：2.7 问题相关组件/Related components：KIE
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

请问一下，我想使用KIE进行关键信息抽取，但是对不同抽取出来的关键信息区域使用不同的识别模型来进行识别。
请问这块应该修改哪块的代码，如何指定不同的识别模型分别识别不同的区域？

**Comment by chenhao86** [comment]
直接把 感兴趣的区域，训练到 一个模型里面，然后通过 识别后的标签不同，走不同的代码。

**Comment by zz1988** [comment]
但是这样就识别了两次，耗时更久，除非是关键信息提取这个模型只根据det的信息来进行分类。

**Comment by chenhao86** [comment]
我是直接用yolo ，把区域抓出来，然后根据标签的不同 ，进行ocr。
